idx: 450
erasor: 
    max_range: 80.0
    num_rings: 20
    num_sectors: 108
    min_h: -1.3 # [m] Note that it depends on the distance between body frame and ground contact point of mobile robots
    max_h: 3.0 # [m] Note that it depends on the distance between body frame and ground contact point of mobile robots
    th_bin_max_h: 0.2 # [m]
    scan_ratio_threshold: 0.2 # The Larger, the more aggressive!!
    minimum_num_pts: 6
    rejection_ratio: 0
    gf_dist_thr: 0.25
    gf_iter: 3
    gf_num_lpr: 20
    gf_th_seeds_height: 0.5
    map_voxel_size: 0.2
    version: 3 # 2: R-GPF / 3: R-GPF w/ blocking (not in paper)


MapUpdater:
    data_name: "00"
    initial_map_path: "/media/shapelim/UX960NVMe1/deep_express/kitti_rosbag/00_0_to_4540_w_interval2_voxel_0.200000.pcd"
    env: "outdoor"
    save_path: "/media/shapelim/UX960NVMe1/deep_express/kitti_rosbag/output"
    query_voxel_size: 0.2
    map_voxel_size: 0.2  
    voxelization_interval: 2
    removal_interval: 4

tf:
     lidar2body: [0.0, 0.0, 1.73, 0, 0.0, 0.0, 1.0] # xyz q_x, q_y, q_z, q_w in order

verbose: true

